src/menu-list.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const React = require('react');
+const mdl = require('material-design-lite/material.min');
+const classnames = require('classnames');
+
+const baseClasses = {
+  'mdl-menu': true,
+  'mdl-js-menu': true
+};
+
+class MenuList extends React.Component {
+ render(){
+  const {
+    className,
+    children,
+    ripple,
+    topLeft,
+    topRight,
+    bottomRight,
+    htmlFor
+  } = this.props;
+
+  const classes = classnames(baseClasses, {
+    'mdl-js-ripple-effect': ripple,
+    'mdl-menu--top-left': topLeft,
```

Comment:
considering using a singular property for this called `position` that takes a string of `top-left`, `top-right`, `bottom-right` or `bottom-left`... we could even normalize `topLeft`, etc if we want